lib/grandpa/grandpa.go

```diff
@@ -232,7 +260,7 @@ func (s *Service) playGrandpaRound() error {
 				log.Trace("[grandpa] failed to check if round is completable", "error", err)
 			}
 
-			finalizable, err := s.isFinalizable(s.state.round - 1)
+			finalizable, err := s.isFinalizable(s.state.round)
```

Comment:
Used to check s.state.round -1, now checking current round.  I don't understand why?

Reply:
if the current round isn't finalizable and the `receiveMessages` function returns, then we won't be able to receive any more votes and the `attemptToFinalizeRound` gets stuck in an infinite loop, I was seeing this happen pretty often, so I changed it